### PR TITLE
Minor fixes in the README

### PR DESCRIPTION
**Pull request by rastamhadi**
Was it me that helped you with the rest of the English in the README? It looks familiar...

### CODE REVIEW

**Comment by vividmuimui** [comment]
Brakeman analysis of 4e6935122de58833dc11ef0dd9eb9cd5809d6777 failed.
Visit the [analysis results page](https://sider.review/gh/repos/93866419/pulls/42) to review [error log](https://sider.review/gh/repos/93866419/pulls/42/logs/780f3536-2916-4ed2-b33c-822fd530818d).

<sub>posted by Sider</sub>

File: README.md [hunk 1]
```diff
@@ -147,4 +147,4 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ## Code of Conduct
 
-Everyone interacting in the LockDiff project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/[USERNAME]/lock_diff/blob/master/CODE_OF_CONDUCT.md).
+Everyone interacting in the LockDiff project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/vividmuimuij/lock_diff/blob/master/CODE_OF_CONDUCT.md).
```
Comment (vividmuimui) [inline README.md]:
```suggestion
Everyone interacting in the LockDiff project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/vividmuimui/lock_diff/blob/master/CODE_OF_CONDUCT.md).
```

**Comment by vividmuimui** [comment]
> Was it me that helped you with the rest of the English in the README? 

Yes! That's right! 
🙏

File: README.md (hunk 1 above)
Comment (rastamhadi) [reply README.md]: oops

**Comment by vividmuimui** [comment]
Brakeman analysis of f3807ccb297cdf2a353e4668decf851f7603eac8 failed.
Visit the [analysis results page](https://sider.review/gh/repos/93866419/pulls/42) to review [error log](https://sider.review/gh/repos/93866419/pulls/42/logs/3a600d35-d573-4c04-9f8c-1746988f4f01).

<sub>posted by Sider</sub>

**Pull request merged**